src/Commands/CmdSession.cpp

```diff
@@ -31,6 +31,8 @@
 #include "ProgressDlg.h"
 #include "SysInfo.h"
 #include "AppUtils.h"
+#include "MainWindow.h"
+
```

Comment:
commands should not have access to the main window.
Instead, extend the OpenFile() method of the ICommand interface.

Reply:
This was an oversight. Fixed.